src/ansys/mapdl/core/cndparser.py

```diff
@@ -0,0 +1,248 @@
+#**********************************************************************
+# Importing  used libraries
+#**********************************************************************
+import sys
+import os    
+from enum import Enum
+from decimal import Decimal
+import copy
+import numpy as np
+import xlsxwriter
+import matplotlib.pyplot as plt
+def cndparser( filename, exclflg, contsrno, columinfo):
+    """1. Reading a cnd file and storing the data in list and arrays.
+       2. Writing the contact information in an excel sheet if requested (excflg = 1).
+       3. Plotting a contact information for a contact pair if requested.  
+        Parameters
+        ----------
+        filename : str
+            Contact diagnosis input file with .cnd extention.
+        exclflg  : int, in [0,1]
+            if 1  spread sheet is generated
+        contsrno : int, optional
+            cont pair serial number for which a time evolution plot is requested.  
+        columinfo: str, optional
+            Short name of the column need to be plotted as lited in (NLHIST,CONT)            
+    """
+    if not (os.path.exists(filename)):
+        sys.exit('The file, ' + filename +' does not exist.')     

```

Comment:
Should simply raise FileNotFoundError here rather than doing a full python exit.

```suggestion
    if not (os.path.exists(filename)):
        raise FileNotFoundError(f'The file {filename' does not exist.')

```